epochStart/bootstrap/shardStorageHandler.go

```diff
@@ -140,86 +141,97 @@ func (ssh *shardStorageHandler) SaveDataToStorage(components *ComponentsNeededFo
 		return err
 	}
 
-	err = ssh.saveTries(components)
+	err = ssh.commitTries(components)
 	if err != nil {
 		return err
 	}
 
 	return nil
 }
 
-func (ssh *shardStorageHandler) getProcessMiniBlocks(
-	pendingMiniBlocks map[string]*block.MiniBlock,
-	meta *block.MetaBlock,
-	headers map[string]data.HeaderHandler,
-) ([]bootstrapStorage.MiniBlocksInMeta, error) {
-	processedMiniBlocks := make([]bootstrapStorage.MiniBlocksInMeta, 0)
-	for _, epochStartShardData := range meta.EpochStart.LastFinalizedHeaders {
-		if epochStartShardData.ShardID != ssh.shardCoordinator.SelfId() {
+func getEpochStartShardData(metaBlock *block.MetaBlock, shardId uint32) (block.EpochStartShardData, error) {
+	for _, epochStartShardData := range metaBlock.EpochStart.LastFinalizedHeaders {
+		if epochStartShardData.ShardID != shardId {
```

Comment:
Perhaps switch to positive condition and return?

Reply:
done